karabo_bridge/simulation.py

```diff
@@ -81,6 +87,13 @@ def corr_passport(self):
         ]
         return passport
 
+    @property
+    def data_shape(self):
+        if self.modules == 1:
+            return (self.mod_y, self.mod_x, self.pulses)
+        else:
+            return (self.pulses, self.modules, self.mod_x, self.mod_y)
```

Comment:
How many of the possible cases (single/combined, raw/corrected) have we checked here?

Reply:
- single + raw
- combined + raw
- combined + corrected
What would be missing for completion is single + corr

Reply:
OK, going back a bit on what I said before, maybe we shouldn't offer single + corrected data from the simulator until we've examined some real data to check what it looks like. But in that case it should fail with a clear error if single + corrected is what's requested.

Reply:
I've just added that